backend/src/main/java/ca/gov/dtsstn/cdcp/api/web/v1/controller/ConfirmationCodesController.java

```diff
@@ -41,18 +42,21 @@ public ConfirmationCodesController(UserService userService) {
 	}
 
 	@GetMapping
+	@ApiResponse(responseCode = "405", description = "Method Not Allowed")
```

Comment:
Since other methods are being annotated with Spring's `@ResponseStatus` (instead of Swagger's `@ApiResponse):

```suggestion
	@ResponseStatus(HttpStatus.METHOD_NOT_ALLOWED)
```

Reply:
It makes sense to add both sometimes.. in this case, just the `@ResponseStatus` should be enough.

Reply:
fixed